Index.html

```diff
@@ -93,9 +93,9 @@ <h3>Youtube Video</h3>
         </p>
         <!--Mail function needs mail app, find later.-->
       <picture>
-        <source type="image/svg+xml" srcset="Images/sc-logo.svg" media="(max width:70px)">
-        <img src= "Images/sc-logo.png"
-        srcset="Images/sc-logo@1.5x.png 1.5x, Images/sc-logo@2x.png 2x">
+        <source type="image/svg+xml" srcset="Images/sc-logo.svg">
+        <img src= "Images/sc-logo.png" width="70" alt="Sault College logo"
+        srcset="Images/sc-logo.png, Images/sc-logo@1.5x.png 1.5x, Images/sc-logo@2x.png 2x">
```

Comment:
just use the width attribute on `<img>`
don't forget to include the main 1x image in the srcset